### PR TITLE
[N/A] Updated Readme

### PR DESCRIPTION
**Pull request by pjbroadbent**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,73 +2,76 @@
...
 
 ## Overview
-
 OpenFin FDC3 API uses the message Channels framework to expose its API to consuming applications.  You can see the documentation for these APIs here:  https://cdn.openfin.co/docs/javascript/stable/InterApplicationBus.Channel.html.
 
-This project consist of 2 parts:
-1. The FDC3 Provider, taking care of intents, context and resolving them (UI)
+This project consist of 3 parts:
+1. The FDC3 Service, taking care of intents, context and resolving them (UI)
 2. The FDC3 Client, exposing API's for applications to handle/raise intents with contexts
```
Comment (bryangaleopenfin) [inline README.md]: This should just be APIs, right?

File: README.md [hunk 2]
```diff
@@ -2,73 +2,76 @@
...
-This is a WIP living implementation of the FDC3 API.
-
-### Usage
-
-An in-depth usage guide and additional documentation will be published in due course.
-
-## Run Locally
-
-To preview the functionality of the service without integrating it into an existing application - or to start contributing to the service - the service can be ran locally. By checking out this repo and then running the project.
+[Official API spec](hhttps://fdc3.finos.org/)
```
Comment (bryangaleopenfin) [inline README.md]: hhttps

File: README.md [hunk 3]
```diff
@@ -2,73 +2,76 @@
...
 
 ## Getting Started
 
-Using the FDC3 service is done in two steps, add the service to application manifest and import the API:
+Integrating the FDC3 service is done in two steps, add the service to application manifest and import the API:
```
Comment (bryangaleopenfin) [inline README.md]: "Integrating the FDC3 service is done in two steps - add the service to application manifest, and import the API:"

File: README.md [hunk 4]
```diff
@@ -2,73 +2,76 @@
...
 To ensure the service is running, you must declare it in your application config.
 
 ```
-"services": [
-    {
-        "name": "fdc3"
-    }
+"services":
+[
+   {"name":"fdc3"}
```
Comment (bryangaleopenfin) [inline README.md]: Space after the ':'?

File: README.md (hunk 2 above)
Comment (pjbroadbent) [reply README.md]: Also moved this to first section

File: README.md (hunk 3 above)
Comment (pjbroadbent) [reply README.md]: Aligned punctuation with layouts

**Review by bryangaleopenfin — APPROVED** [review]

**Review by tomrobertsOF — APPROVED** [review]

**Pull request merged**